source/server-selection/server-selection.rst

```diff
@@ -521,6 +527,25 @@ Eligibility MUST be determined from ``tag_sets`` as follows:
 - If the ``tag_sets`` list is not empty and no tag set in the list matches any
   candidate server, no servers are eligible servers.
 
+hedge
```

Comment:
All the other readPreference options are configurable through the connection string (`maxStalenessSeconds`, `readPreference`, `readPreferenceTags`). Should we add a new URI option for `hedge` maybe called `readPreferenceHedge`?

Reply:
Since there's already a conversation about this in [DRIVERS-723](https://jira.mongodb.org/browse/DRIVERS-723), let's continue it there rather than this comment thread.